Reorg customize config section, add example config and version field … (#1493)
Maistra/istio-operator: bundle/README.md

1. When `State: Healthy` appears in the `Status` column, Istio is successfully deployed.

## Selecting the Istio version

> The `version` field of the `Istio` custom resource definition defines which version of Istio can be deployed. For a list of available versions, see the [versions.yaml](../versions.yaml) file. This can be set using the `Istio Version` when creating a new `Istio` with the OpenShift Container Platform web console.

Reviewer: Users can also see the list of versions by running `kubectl explain istio.spec.version`.

Author: Thanks, I'll add that too.

- [CNI parameters](https://artifacthub.io/packages/helm/istio-official/cni?modal=values)
- [ZTunnel parameters](https://artifacthub.io/packages/helm/istio-official/ztunnel?modal=values)

## Validating the configuration

> The Operator validates the contents of the `spec.values` field. When the Operator parses this field, and encounters an unknown value, it generates an error message. To force the Operator to pass an unknown value, use the `spec.rawValues` field.

Reviewer: ```suggestion
The Kubernetes API server validates the contents of the `spec.values` field. When it encounters an unknown value, it generates an error message. To pass an unknown value, use the `spec.rawValues` field.
```

> You can use the `oc explain` command to display the valid values for the fields in the Istio Custom Resoure Definition (CRD) file. For example, `oc explain istio.spec.values`.

Reviewer: Should we use `kubectl` instead of `oc` in all of our instructions, since we support vanilla k8s?

Reviewer 2: @luksa based on today's team discussion, I created OSSM-5431 to change oc commands to kubectl commands.
